File: fix_gas.py
import urllib.request
import json
url = 'https://script.google.com/macros/s/AKfycbydXZuGZWqI1rpx0fPJawHMzYekVubxeBCLs9taZPG3glPaFVD19CK7BRx1PZcCkRLf/exec'
req = urllib.request.Request(url)
with urllib.request.urlopen(req) as response:
    data = json.loads(response.read().decode())
    
data['entries'] = [e for e in data.get('entries', []) if e.get('id') is not None]
payload = json.dumps({'action': 'save', 'db': data}).encode('utf-8')
req = urllib.request.Request(url, data=payload, headers={'Content-Type': 'text/plain'})
with urllib.request.urlopen(req) as response:
    print(response.read().decode())

File: fix_gas.js
const fs = require('fs');
async function run() {
    const url = 'https://script.google.com/macros/s/AKfycbydXZuGZWqI1rpx0fPJawHMzYekVubxeBCLs9taZPG3glPaFVD19CK7BRx1PZcCkRLf/exec';
    let res = await fetch(url);
    let data = await res.json();
    data.entries = data.entries.filter(e => e.id !== null);
    
    let payload = { action: 'save', db: data };
    let postRes = await fetch(url, {
        method: 'POST',
        headers: { 'Content-Type': 'text/plain' },
        body: JSON.stringify(payload)
    });
    console.log(await postRes.text());
}
run();
